docs/cli-input/cluster/e3s-load-balancer.json

```diff
@@ -3,7 +3,7 @@
     "SecurityGroups": [
         "{e3s-sg-id}"
     ],
-    "Scheme": "internet-facing",
+    "Scheme": "internet-facing | internal",
```

Comment:
I'd recommend to keep in this place single `internet-facing` value to make json workable by default and move Note about possibility to use internal below...